### DISCUSSION TITLE
Port label background rect moves on scale

### DISCUSSION THREAD

**Discussion by Biowavix** [clientIO/joint#2025] [Q&A]
Hello, I am trying to put a background rectangle to the port labels text, I use [this](https://codepen.io/djimenez-developair/pen/KKBYeQV) demo Wednesday as reference to get it. The problem is that I want to do this for elements that scale (concretly using FreeTransform plugin). I have added an event at the end of the codepen (line 462) so that when an element is clicked it scales it (the amount apparently does not matter) and as you can see, after scaling, the background of the label does not adjust correctly.

Any idea how I can do this? Ideally using FreeTransform plugin.

Thank you in advance!

**Comment by kumilingus** [comment]
It's a bug. Please watch https://github.com/clientIO/joint/issues/2026 from now on.
